src/components/Accounts/Actions/WarningModal.js

```diff
@@ -173,7 +173,7 @@ class WarningModal extends React.Component {
     };
 
     const values = Object.values(checkedAccounts);
-    const checkedClosed = values.filter(a => a.status.name === 'Closed');
+    const checkedClosed = values.filter(a => a.status && a.status.name === 'Closed');
```

Comment:
By the way, I suggest using `.some` also here. As I see in the condition below, `checkedClosed.length > 0` the count is just needed to make some decision. So, in this case finding all closed loans is overhead. [Approach](https://github.com/folio-org/ui-users/pull/1104/files#r361674281) which @SergiyVSergiyenko has proposed, sounds more proper to me.